apps/epic-web/src/pages/index.tsx

```diff
@@ -74,8 +82,8 @@ const Index: NextPage<{
         width: 132,
         height: 112,
       },
-      title: product.title as string,
-      description: product?.description,
+      title: commercePropsProduct?.name as string,
+      description: commercePropsProduct?.description,
```

Comment:
for the redeem dialog we want to use the restricted to product not the default product for the site